helm/cluster-openstack/values.yaml

```diff
@@ -0,0 +1,34 @@
+name: test1
+description: Test cluster using a shared ClusterClass.
+organization: giantswarm-2
+className: v20.0.0-alpha1
+kubernetesVersion: 1.20.9
+releaseVersion: v20.0.0-alpha1
```

Comment:
Shouldn't this be hardcoded rather than configurable? As I see it, the release name is the version of the chart.

Reply:
I wasn't sure if we were going to use other releases or if we consider `v20.0.0-alpha1` to be the last "traditional" release and the rest come from the version of the chart. Hardcoding it makes sense though.

Reply:
In my view v20.0.0-alpha1 was a transitional release and we can start over at 0.1.0 with CAPI (at least for OpenStack). @giantswarm/team-honeybadger any thoughts?